index.js

```diff
@@ -3656,6 +3656,40 @@
       [Fn(a, b), m(a), m(a)],
       Z.sortBy);
 
+  //# zipWith :: (a -> b -> c) -> Array a -> Array b -> Array c
+  //.
+  //. Creates a new array out of the two supplied arrays
+  //. by applying the given function to each equally-positioned
+  //. pair of elements from each given array. Returns the result
+  //. of combining, pairwise, the given arrays using the given
+  //. binary function. The length of the resulting array is equal
+  //. to the length of the shorter input array.
```

Comment:
The first and second sentences are duplicative. This may be sufficient:

```javascript
  //. Returns the result of combining, pairwise, the given arrays using the
  //. given binary function. The length of the resulting array is equal to the
  //. length of the shorter input array.
```

Reply:
Gotcha!